dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -497,6 +495,8 @@ Sedp::init(const RepoId& guid,
   }
 #endif
 
+  max_type_lookup_service_reply_period_ = disco.config()->max_type_lookup_service_reply_period();
```

Comment:
Why is this line changing?

Reply:
I moved it from above, where it was in the middle of writers /  readers setup.